src/gui/embed.cpp

```diff
@@ -39,15 +42,70 @@ auto loadPixmap(const QString& name, int width, int height, const char* const* x
 	if (xpm) { return QPixmap{xpm}; }
 
 	const auto resourceName = QDir::isAbsolutePath(name) ? name : "artwork:" + name;
-	auto reader = QImageReader{resourceName};
-	if (width > 0 && height > 0) { reader.setScaledSize(QSize{width, height}); }
 
-	const auto pixmap = QPixmap::fromImageReader(&reader);
-	if (pixmap.isNull()) {
-		qWarning().nospace() << "Error loading icon pixmap " << name << ": " << reader.errorString();
-		return QPixmap{1, 1};
+	qreal devicePixelRatio = QGuiApplication::primaryScreen()->devicePixelRatio();
```

Comment:
This value is only needed if we're loading an SVG it should be moved into that condition.